src/main/java/io/kontur/disasterninja/controller/MetricsController.java

```diff
@@ -0,0 +1,58 @@
+package io.kontur.disasterninja.controller;
+
+import io.kontur.disasterninja.dto.UserMetricDto;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.prometheus.PrometheusMeterRegistry;
+import io.prometheus.client.CollectorRegistry;
+import io.prometheus.client.Summary;
+import io.swagger.v3.oas.annotations.Operation;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.CollectionUtils;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/rum")
+public class MetricsController {
+
+    private final Summary summary;
+
```

Comment:
I'm not sure if it can help to get rid of constructor in a Controller class, as you recieve metrics from a request and parse them then register, but while I was working with metrics for Grafana I found it very useful to use io.micrometer:micrometer-core:VERSION:io.micrometer.core.annotation package. Maybe it could simplify this code snippet, but I'm not sure in this case

Reply:
Nope. Not in the case when we need to pass value that we gathered from outside